pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -111,6 +111,13 @@ var adminCommands = []*cli.Command{
 		Name:   "reset-admin-password",
 		Usage:  "reset-admin-password <new password>",
 		Action: runDbCommand(resetPasswordCommand),
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "passwordFromStdin",
```

Comment:
Having discussed internally, we would like to harmonize with grafana-server and migrate to kebab-case flags in grafana-cli:
```suggestion
				Name:  "--password-from-stdin",
```